pyproject.toml

```diff
@@ -57,18 +57,20 @@ classifiers=[
 python = ">=3.8,<4.0"
 requests = "^2.31.0"
 pyyaml = ">=5.4"
-loguru = "0.5.0"
+loguru = "0.5.3"
 typing-extensions = ">=3.7.4"
 dnspython = ">=2.0.0"
 python-dateutil = ">=2.5.3"
-urllib3 = ">=1.21.1"
+urllib3 = "1.25.3"
 tqdm = ">=4.64.1"
 numpy = ">=1.22.0"
 grpcio = ">=1.44.0"
 grpc-gateway-protoc-gen-openapiv2 = "0.1.0"
 googleapis-common-protos = ">=1.53.0"
 lz4 = ">=3.1.3"
 protobuf = "~=3.20.0"
+aenum = "3.1.11"
+pydantic = "1.10.12"
```

Comment:
These are new dependencies required by the openapi generator. Both have permissive open source licenses, so no issue there.